src/averageAges.js

```diff
@@ -15,7 +15,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
+  const filteredMen = people
+    .filter(person => person.sex === 'm'
+      && (century ? Math.ceil(person.died / 100) === century : true));
+
+  const sumOfAge = filteredMen
+    .map(man => man.died - man.born)
+    .reduce((a, b) => a + b);
+
+  return sumOfAge / filteredMen.map(man => man.died - man.born).length;
```

Comment:
Do you really need `.map` here? length won't change
```suggestion
  return sumOfAge / filteredMen.length;
```